src/components/projects/new-project.jsx

```diff
@@ -39,12 +46,17 @@ export default function NewProject({ isOpen, onOpen, onClose }) {
       })
   }
 
-  const stringIsNotEmpty = (value) => {
+  const stringIsNotEmpty = value => {
     if (!value?.length > 0) {
       return 'Value must not be empty!'
     }
   }
 
+  const validateProjectName = value => {
+    setProjectNameValue(value)
```

Comment:
can you set this to be value.trim() and reuse projectNameValue so you don't have to call trim() repeatedly